### ISSUE TITLE
DungeonTools conflicts with MDT

### ISSUE THREAD

**Issue by mbelsky** [Tomslack/ManbabyDungeonTools#65]
Hey,

I have both DungeonTools and MDT installed and it looks like they doesn't work together.

https://user-images.githubusercontent.com/3923527/103372122-3b20c600-4ae2-11eb-9a4c-04e59372418e.mp4

And if I disable MDT then DungeonTools works well. May be you have any idea about reasons of that.

DungeonTools v1.1.6

**Comment by srenauld** [comment]
Yeah; same global variable. The main reason it was kept was to not break compatibility with any third-party integrations ([mdt-guide](https://www.curseforge.com/wow/addons/mdt-guide) being a good example; it was too short notice.

As a result, for now, I'm afraid it's one or the other 😢

**Comment by ArPharazon** [comment]
Uninstall MDT and use DungeonTools from now on.

**Comment by mbelsky** [comment]
> Uninstall MDT and use DungeonTools from now on.

Yea, I already did that. I'm just worried new users will stuck with that issue.

**Comment by srenauld** [comment]
I'm moving all of it to a different namespace at the moment, so people can run both; also building an adapter so DT can read routes stored in MDT.

Should be done by EOD today, depending on how keys go.
